contract/contracts/PixelAvatars.sol

```diff
@@ -37,8 +41,11 @@ contract PixelAvatars is
         // mintPrice = 0.01 ether;
     }
 
-    function _baseURI() internal view virtual override returns (string memory) {
-        return baseURI;
+    /**
+     * @param amount The amount of royalties to be set.
```

Comment:
I personally would use single line natspec comment and remove the `.` at the end here for consistensy.
`/// @param amount The amount of royalties to be set`

Reply:
Gotcha

Reply:
It does actually save up on the final size of the smart contract, right? And saving a little bit of them gas fees on deployment.